argo/workflow/titan-bank.yaml

```diff
@@ -12,7 +12,7 @@ spec:
       - name: image_version
         value: release-4.0-nightly
       - name: storage_class
-        value: local-storage
+        value: gp2
```

Comment:
what's gp2?

Reply:
gp2 is the storage class name we use in AWS. This can be overwritten to other values when we run this workflow in other clusters.